uber/site_sections/preregistration.py

```diff
@@ -8,33 +7,57 @@ def to_sessionized(attendee, group):
     else:
         return Charge.to_sessionized(attendee)
 
+
 def check_prereg_reqs(attendee):
-    if attendee.badge_type == PSEUDO_DEALER_BADGE and not attendee.cellphone:
+    if attendee.badge_type == c.PSEUDO_DEALER_BADGE and not attendee.cellphone:
         return 'Your phone number is required'
-    elif attendee.amount_extra >= SHIRT_LEVEL and attendee.shirt == NO_SHIRT:
+    elif attendee.amount_extra >= c.SHIRT_LEVEL and attendee.shirt == c.NO_SHIRT:
         return 'Your shirt size is required'
 
+
 def check_dealer(group):
```

Comment:
As indicated by the fact that I had to put `COLLECT_INTERESTS` here a couple times, these validations should definitely not be hardcoded like this. Any chance we can easily move them to the new format?

I know dealers aren't a class defined in models.py, but it's a very common paradigm - and I expect conventions will want other add-on definitions with their own validations (like artists in artists alley, staff, guests). What're the chances that we can build in support for validation on psuedo-classes defined by parameters like table ownership and badge type?